posts/2020-10-29-macos-arm64.rst

```diff
@@ -141,14 +141,14 @@ For autotools package, add the following to ``recipe/meta.yaml``,
 
    requirements:
      build:
-       - libtool   # [unix]
+       - gnuconfig   # [unix]
 
 and to ``recipe/build.sh``,
 
 .. code-block:: bash
 
    # Get an updated config.sub and config.guess
-   cp $BUILD_PREFIX/share/libtool/build-aux/config.* .
+   cp BUILD_PREFIX/share/gnuconfig/config.* .
```

Comment:
Wow so this was just in? Seems to work fine. Though I wonder why to remove the dollar sign...?

Reply:
that was a mistake. Can you send a PR?